README.md

```diff
@@ -90,15 +97,20 @@ To start reading from the Safety laser scanner and publishing complete scans exe
 If you wish to set parameters from the command line, add them to the end of the command as follows: `parameter:=value`, separated by spaces.
 
 ```bash
-roslaunch psen_scan psen_scan.launch host_ip:=192.168.0.20 host_udp_port:=3050
+roslaunch psen_scan psen_scan.launch sensor_ip:=192.168.0.10 host_ip:=192.168.0.20 host_udp_port:=3050
 ```
-This example configures the Safety laser scanner to send it´s frames to 192.168.0.20:3050.
+This example configures the Safety laser scanner at 192.168.0.10 to send it´s frames to 192.168.0.20:3050.
+
+### Adjust expert parameters
+If you've created an application package with your own launch file as described in the
+[tutorials](http://wiki.ros.org/psen_scan/Tutorials/),
+you can easily adjust the aforementioned configuration parameters.
 
 ## You need further information?
 Our international hotline staff will support you individually about our ROS packages at
 ros@pilz.de
 
-Find more information about the Pilz manipulator module on the [product website](https://www.pilz.com/en-INT/eshop/00106002197131/PSENscan-Safety-Laser-Scanner).
+Find more information about the Pilz Safety laser scanner on the [product website](https://www.pilz.com/en-INT/eshop/00106002197131/PSENscan-Safety-Laser-Scanner).
```

Comment:
Maybe use lower-case? Or same as in headline?
```suggestion
Find more information about the Pilz safety laser scanner on the [product website](https://www.pilz.com/en-INT/eshop/00106002197131/PSENscan-Safety-Laser-Scanner).
```

Reply:
Solved in 01dcea65f79ebae8a82975a9271cb7ec29a87550